pom.xml

```diff
@@ -1,10 +1,10 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.sunbird</groupId>
-  <artifactId>user-service</artifactId>
+  <artifactId>sunbird-user-org-service</artifactId>
```

Comment:
Lets simply call as "user-org-service". Org.sunbird is the groupId and so lets not repeat it.

Reply:
done